Makefile

```diff
@@ -31,18 +31,18 @@ clean: ## Cleanup
 .PHONY: lib src test doc
 
 lib: ## Build libs
-	$(MAKE) -C lib
+	+$(MAKE) -C lib
```

Comment:
Are these not equivalent?  From https://www.gnu.org/software/make/manual/make.html#Recipes:

> As a special feature, using the variable MAKE in the recipe of a rule alters the effects of the ‘-t’ (‘--touch’), ‘-n’ (‘--just-print’), or ‘-q’ (‘--question’) option. Using the MAKE variable has the same effect as using a ‘+’ character at the beginning of the recipe line. See Instead of Executing the Recipes. This special feature is only enabled if the MAKE variable appears directly in the recipe: it does not apply if the MAKE variable is referenced through expansion of another variable. In the latter case you must use the ‘+’ token to get these special effects.

Reply:
It doesn't hurt if it is there. Should we ever consider replacing the `$(MAKE)` with something else (maybe `BEES_MAKE=$(MAKE) --another-flag`) we are already prepared.

Reply:
It seems we both learned new Makefile syntax today.

Reply:
Actually, I figured out a lot of new stuff for Makefiles during working on my wine-proton branch... :-)

But I also figured out I might want to look into meson/ninja a little more.